lib/widgets/screens/sign_in/sign_in_screen.dart

```diff
@@ -64,161 +65,342 @@ class _SignInScreenState extends State<SignInScreen> {
+                                                'Account not found. Please check your credentials and try again or create a new account'),
+                                          ),
+                                        );
+                                        if (context.mounted) {
+                                          emailController.text = '';
+                                          passwordController.text = '';
+                                        }
+                                      } else if (signInError ==
+                                          'passwordNoMatch') {
+                                        scaffoldManager
+                                            .showSnackBar(const SnackBar(
                                           content: Text(
-                                              'Account not found. Please check your credentials and try again or create a new account'),
-                                        ),
-                                      );
-                                      if (context.mounted) {
-                                        emailController.text = '';
+                                              'Please enter you correct password'),
+                                        ));
                                         passwordController.text = '';
                                       }
-                                    } else if (signInError ==
-                                        'passwordNoMatch') {
-                                      scaffoldManager
-                                          .showSnackBar(const SnackBar(
-                                        content: Text(
-                                            'Please enter you correct password'),
-                                      ));
-                                      passwordController.text = '';
+                                    } else {
+                                      router.push('/');
                                     }
-                                  } else {
-                                    router.push('/');
                                   }
-                                }
-                              },
-                              child: Padding(
-                                padding: const EdgeInsets.all(8.0),
-                                child: Text(
-                                  'Sign In',
-                                  style: GoogleFonts.poppins(
-                                    fontSize: 14,
-                                    fontWeight: FontWeight.bold,
+                                },
+                                child: Padding(
+                                  padding: const EdgeInsets.symmetric(
+                                    vertical: Insets.widgetSmallInset,
+                                    horizontal: Insets.widgetLargeInset * 2.5,
+                                  ),
```

Comment:
We should avoid performing math operations on constants that will result in another constant. This might make sense, for example, when one widget is sized relative to another, but in this case 2.5xlargeInset seems like arbitrary value.

Reply:
Thanks for that comment. I'll create a new constant.

Reply:
Made changes by creating new constants.